io/eolearn/io/local_io.py

```diff
@@ -161,7 +164,12 @@ def execute(self, eopatch, *, filename=None):
         :return: Unchanged input EOPatch
         :rtype: EOPatch
         """
-        feature_type, feature_name = next(self.feature(eopatch))
+        try:
+            feature_type, feature_name = next(self.feature(eopatch))
+        except ValueError as error:
+            LOGGER.warning(error)
```

Comment:
is this error explicit enough for the user to understand that the feature is missing? 

or should we add a string that does that?

Reply:
Yes, the error is very explicit. Example: 

> WARNING eolearn.io.local_io:170: Feature VALID_COUNT_MULTI_150m of type FeatureType.MASK_TIMELESS was not found in EOPatch

Reply:
perfect